src/client/components/map/index.ts

```diff
@@ -47,6 +49,32 @@ const filteredLabelLayers = [
   "poi-label"
 ];
 
+export function getChloroplethSteps(metricKey: string) {
+  const compactnessSteps = [
+    [0.3, "#edf8fb"],
+    [0.4, "#b2e2e2"],
+    [0.5, "#66c2a4"],
+    [0.6, "#2ca25f"],
+    [0.6, "#006d2c"]
```

Comment:
I think the last stop should be up to 1 like so (without this my 78% compactness district wasn't showing up as dark green):
```suggestion
    [1.0, "#006d2c"]
```